testfiles/2ZoneDataCenterHVAC_wEconomizer.idf

```diff
@@ -2889,6 +2890,7 @@
   ElectricEquipment:ITE:AirCooled,
     EastDataCenter_Equip,    !- Name
     East Zone,               !- Zone Name
+	,
```

Comment:
Actually, it would be good if one zone in this example file used the new model.  Or add a new example file that uses it.